content/docs/command-reference/plots/diff.md

```diff
@@ -48,7 +48,6 @@ Another way to display plots is the `dvc plots show` command, which just lists
 all the current plots, without comparisons.
 
 [certain metrics]: /doc/user-guide/visualizing-plots#supported-plot-file-formats
-
 [plot templates]:
```

Comment:
The linter fixed this during a merge of `main` (to solve conflicts). Merging in...